handlers/filetest.py

```diff
@@ -0,0 +1,15 @@
+from datetime import date
+import os
+
+def create_feedback_file(intentname,request):
+   today = date.today()
+   cwd = os.getcwd()
+   files = os.listdir(cwd)
+   print(files)
+   fname =  './nlp_intern/handlers/feedback/' + intentname + "/" + today.strftime("%d%m%y")
+   f = open(fname + ".txt",'a')
+   f.write(request.text + '\n')
```

Comment:
since we are using only `request.text`.. is it necessary for the entire `request` object to be passed as argument